canonical-web-and-design/cn.ubuntu.com
# added new OTA10 news article

# Details
- card: https://trello.com/c/wEqto7j7
- copy doc: https://docs.google.com/document/d/1z-YUlXj5GGazP7mA4NtJVr0l-RaU_LEYShCIrRNQBtk/edit#heading=h.8gz6tvb7ycj5
## done
- optimised all the images
- created news page
- added article to index
## qa
- look at the index page /news/ to see that the new post is top of the phone list
- look at the news article /news/2016/04/08/phone-ota10/ and see that it looks like the copy doc and the images work for all sizes... not sure I did them right
